app/src/App.tsx

```diff
@@ -40,7 +39,6 @@ const App: React.FC = () => {
         {!status ? (
           <>
             <HeaderNoRouter />
-            <SettingsViewContainer networkId={networkId} />
```

Comment:
The use of this was that when rpc node is down settings view would popup and you would have to select other node when default one or one selected is down....I don't see that handled anywhere? This is what i get if node is down. It should display the modal you made...
![Screenshot 2021-05-12 at 12 30 52](https://user-images.githubusercontent.com/33226956/117961502-7b107900-b31e-11eb-8821-a257606812b4.png)

Reply:
![Screenshot 2021-05-12 at 12 57 45](https://user-images.githubusercontent.com/33226956/117964238-ab0d4b80-b321-11eb-8fe7-592b6dfa8c83.png)
Its the last things in specification.... geronimo wrote it more clearly.......